for no reason, catkin wants this to be "standard" with another generated one

--- ros_comm/CMakeLists.txt.orig	2020-02-21 01:13:08.000000000 +0100
+++ ros_comm/CMakeLists.txt	2020-03-17 15:58:48.666524517 +0100
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 2.8.3)
 project(ros_comm)
 find_package(catkin REQUIRED)
 catkin_metapackage()
